src/multi-level-checkbox-editor/multi-level-checkbox-editor.tsx

```diff
@@ -12,20 +16,42 @@ interface Props {
     nodes: Array<Node>;
     onNodesChange: (nodes: Array<Node>) => void;
     levelHeaderLabels: Array<string>;
+    searchPlaceholder?: string;
+    infoText?: string;
+    noDataText?: string;
+    onFilterChange?: (value: string) => void;
 }
 
 interface State {
     selectedNodeIds: Array<string>;
+    searchValue: string;
 }
 
+const copy = <T extends {}>(o: T) => Object.assign({}, o);
```

Comment:
Would `{...o}` do the same?

Reply:
Spread defines properties, Object.assign sets them and accordingly invoke setters, but in our case It does not matter - both works, It's just difference in verbosity.